app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/shared/cvssScore.js

```diff
@@ -0,0 +1,33 @@
+import { getSeverityColors } from './cvssUtils.js'
+import React from 'react'
+
+/**
+ *
+ *
+ * @param {{
+ *  score: any
+ *  severity: any
+ * }} props
+ */
+export default function CvssScore({ score, severity }) {
+  return (
+    <div
+      className={`p-2 rounded border w-32 ${getSeverityColors(
+        /** @type {number} */ (score)
+      )}`}
+    >
+      {typeof score === 'number' ? (
+        <>
+          <p className="text-center text-2xl">
+            <b>{score}</b>
```

Comment:
A score has always 1 digit after the `.`...
```suggestion
            <b>{score.toFixed(1)}</b>

Reply:
done